.github/workflows/ci-cmake.yml

```diff
@@ -71,6 +73,9 @@ jobs:
       matrix:
         os: [windows-2016, windows-2019]
         build_type: [Debug, Release]
+        shared: ["True", "False"]
```

Comment:
It says `shared: ["True", "False"]` here while it says `shared_libs: [ON, OFF]` on line 25. This is probably intentional, but I just wanted to point out the inconsistency in case it was accidental.

Reply:
Sorry, I don't know.